ash/src/extensions/ext/drm_format_modifier.rs

```diff
@@ -0,0 +1,83 @@
+use crate::prelude::*;
+use crate::vk;
+use crate::{Device, Instance};
+use std::ffi::CStr;
+use std::mem;
+
+#[derive(Clone)]
+pub struct DrmFormatModifier {
+    handle: vk::Device,
+    fp: vk::ExtImageDrmFormatModifierFn,
+}
+
+impl DrmFormatModifier {
```

Comment:
```suggestion
/// <https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/VK_EXT_image_drm_format_modifier.html>

impl DrmFormatModifier {
```

Reply:
Seems kinda odd to put it above the `impl` statement. I would assume the struct would be a better spot?

Reply:
I think I managed to break github :)